emscripten.py

```diff
@@ -1810,13 +1812,14 @@ def create_fp_accessors(metadata):
   var func = Module['asm']['%(original)s'];
   // If there is no wasm function, this may be a JS library function or
   // something from another module.
+  %(originalAccessor)s
```

Comment:
I think this part is already handles above now isn't it?

I think that was what alon fixed in #9810?  Maybe this part of this change can be reverted now?

Reply:
I believe the side exports are added to the `Module` object and not `Module['asm']` thus the line `var func = Module['asm']['%(original)s'];` can only retrieve the original exports from the Main module and not the side. For this PR, we actually need the original functions from the side module, thus the need to retrieve the original side functions from the `Module` object instead  `originalAccessor = ('if (!func) func = Module["_%s"];' % (name))`